Testing/SystemTests/tests/framework/LiquidsReflectometryReductionTest.py

```diff
@@ -114,6 +114,66 @@ def validate(self):
         return self._success
 
 
+class LRReflectivityOutputResolutionTest(systemtesting.MantidSystemTest):
+    """
+        Test the reflectivity output algorithm with resolution calculation
+    """
+    def runTest(self):
+        LiquidsReflectometryReduction(RunNumbers=[190367],
+                                      NormalizationRunNumber=0,
+                                      SignalPeakPixelRange=[142, 152],
+                                      SubtractSignalBackground=True,
+                                      SignalBackgroundPixelRange=[139, 155],
+                                      NormFlag=False,
+                                      NormPeakPixelRange=[142, 151],
+                                      NormBackgroundPixelRange=[139, 154],
+                                      SubtractNormBackground=True,
+                                      LowResDataAxisPixelRangeFlag=True,
+                                      LowResDataAxisPixelRange=[70, 178],
+                                      LowResNormAxisPixelRangeFlag=True,
+                                      LowResNormAxisPixelRange=[70, 178],
+                                      TOFRange=[51978.9, 65266.7],
+                                      IncidentMediumSelected='air',
+                                      GeometryCorrectionFlag=False,
+                                      QMin=0.005,
+                                      QStep=0.02,
+                                      AngleOffset=0.01,
+                                      AngleOffsetError=0.001,
+                                      ApplyScalingFactor=False,
+                                      ScalingFactorFile='',
+                                      SlitsWidthFlag=True,
+                                      CropFirstAndLastPoints=False,
+                                      ApplyPrimaryFraction=False,
+                                      OutputWorkspace='reflectivity_190367')
+
+        output_path = get_file_path('lr_output.txt')
+
+        # Remove the output file if it exists
+        if os.path.isfile(output_path):
+            os.remove(output_path)
+
+        LRReflectivityOutput(ReducedWorkspaces=["reflectivity_190367"],
+                             ComputeDQ=True,
+                             OutputFilename=output_path)
+
+        # Find the dQ/Q value in the output file to determine success
+        self._success = False
+        if os.path.isfile(output_path):
+            with open(output_path, 'r') as fd:
+                for line in fd.readlines():
+                    if line.startswith("# dQ/Q"):
+                        toks = line.split('=')
+                        dq_over_q = float(toks[1])
+                        if abs(dq_over_q - 0.0273679) < 0.00001:
+                            self._success = True
+            os.remove(output_path)
```

Comment:
This would be better done in a `def cleanup(self):` function which the framework will always run